doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1747,69 +1718,67 @@ name of the variable (here :g:`n`) is chosen based on :g:`T`.
              name the variable in the current goal and in the context of the
              existential variable. This can lead to surprising behaviors.
 
-.. tacv:: instantiate (@natural := @term)
-
-   This variant selects an existential variable by its position.  The
+   The second form refines an existential variable selected by its position.  The
    :n:`@natural` argument is the position of the existential variable
-   *from right to left* in the conclusion of the goal.  (Use one of
+   *from right to left* in the goal.  (Use one of
    the variants below to select an existential variable in a
    hypothesis.)  Counting starts at 1 and multiple occurrences of the
    same existential variable are counted multiple times.  Because this
    variant is not robust to slight changes in the goal, its use is
    strongly discouraged.
 
-.. tacv:: instantiate ( @natural := @term ) in @ident
-          instantiate ( @natural := @term ) in ( value of @ident )
-          instantiate ( @natural := @term ) in ( type of @ident )
+   :n:`in @ident`
+     Selects the hypothesis :n:`@ident`.
 
-   These allow to refer respectively to existential variables occurring in a
-   hypothesis or in the body or the type of a local definition (named :n:`@ident`).
+   :n:`in %|- *`
+     Selects all hypotheses and the goal.
 
-.. tacv:: instantiate
+   .. todo PR above is correct (in |- *)?
 
-   This tactic behaves functionally as :tacn:`idtac`.
+   :n:`in ( type of @ident )`
+    Selects existential variables with the type :n:`@ident` in a
+    :term:`local definition <context-local definition>`
 
-   .. deprecated:: 8.16
+   :n:`in ( value of @ident )`
+    Selects existential variables that have type :n:`@ident` in the body of
+    a :term:`local definition <context-local definition>`.
 
 .. tacn:: admit
-   :name: admit
 
    This tactic allows temporarily skipping a subgoal so as to
    progress further in the rest of the proof. A proof containing admitted
    goals cannot be closed with :cmd:`Qed` but only with :cmd:`Admitted`.
 
-.. tacv:: give_up
+   :tacn:`give_up` is a synonym for `admit`.
 
-   Synonym of :tacn:`admit`.
+.. todo PR: is give_up worth keeping?  See proof-mode.rst
```

Comment:
I think it's completely fine to have synonyms. However, the description of the two tactics should be merged. I think the current location of `give_up` is the right one.